src/reducers/real.jsx

```diff
@@ -152,7 +152,6 @@ const realReducer = (state = initialState, action) => {
         }
 
         case SET_REAL_DATA: {
```

Comment:
can we rename this reducer too? since u r renaming everything else?

Reply:
umm i'm so confused by all these names.. @_@;;

Reply:
I'll do a large refactor after the retreat.